sa.sy.bluerecorder.yaml

```diff
@@ -0,0 +1,79 @@
+app-id: sa.sy.bluerecorder
+runtime: org.gnome.Platform
+runtime-version: "3.38"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+cleanup:
+  - /share/man
+  - /share/doc
+  - /share/gtk-doc
+command: /app/opt/blue-recorder
+finish-args:
+  - --filesystem=home
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --talk-name=org.gnome.Shell.Screencast
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - shared-modules/dbus-glib/dbus-glib-0.110.json
+
+  - name: bluerecorder
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/rust-stable/bin
+      env:
+        CARGO_HOME: "/run/build/bluerecorder/cargo"
+        RUST_BACKTRACE: "1"
+    build-commands:
+      - mkdir -p $FLATPAK_DEST/opt $FLATPAK_DEST/share/icons/hicolor/128x128/apps/
+      - install -D flatpak/sa.sy.bluerecorder.metainfo.xml -t $FLATPAK_DEST/usr/share/metainfo/
+      - install -D flatpak/sa.sy.bluerecorder.desktop -t $FLATPAK_DEST/share/applications/
+      - cp -v data/blue-recorder@x128.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/sa.sy.bluerecorder.png
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose 
+      - cargo --offline build --release --verbose
+      - cp target/release/blue-recorder $FLATPAK_DEST/opt 
```

Comment:
this should go in `$FLATPAK_DEST/bin` since it is the executable.

Reply:
Some directory must be placed in the same where executable is. as i know /opt is fine for that